lgc/builder/DescBuilder.cpp

```diff
@@ -403,6 +403,24 @@ Value *DescBuilder::CreateGetBufferDescLength(Value *const bufferDesc, Value *of
   return CreateNamedCall(lgcName::LateBufferLength, getInt32Ty(), {bufferDesc, offset}, Attribute::ReadNone);
 }
 
+// =====================================================================================================================
+// Return the i64 difference between two pointers, dividing out the size of the pointed-to objects.
+// For buffer fat pointers, delays the translation to patch face.
```

Comment:
face -> phase ? (also in the header file)

Reply:
Ah yeah, of course. Whatever I thought when writing this 😅